memory/numa_test.py

```diff
@@ -118,16 +119,21 @@ def test_thp_compare(self):
         """
         Test PFN's before and after offlining
         """
-        thp = "/sys/kernel/mm/transparent_hugepage/enable_thp_migration"
+        thp = "/sys/kernel/mm/transparent_hugepage/enabled"
         if os.path.isfile(thp):
-            self.nr_pages = self.params.get(
-                'nr_pages', default=100)
-            os.chdir(self.teststmpdir)
-            self.log.info("Starting test...")
-            cmd = './bench_movepages -n %s' % self.nr_pages
-            ret = process.system(
-                cmd, shell=True, sudo=True, ignore_status=True)
-            if ret != 0:
-                self.fail('Please check the logs for failure')
+            with open(thp, 'r') as thp_file:
```

Comment:
Please use genio api here to read the file.

Reply:
Addressed the above in latest push